Gemfile

```diff
@@ -1,3 +1,7 @@
 source 'https://rubygems.org'
 
 gemspec
+
+group :test do
```

Comment:
`binding.pry` is not accessible when running specs if not declared in the `test` group. This is why I have added it there.